scalalib/src/mill/scalalib/JavaModule.scala

```diff
@@ -273,7 +274,8 @@ trait JavaModule
    * Default artifact types to fetch and put in the classpath. Add extra types
    * here if you'd like fancy artifact extensions to be fetched.
    */
-  def artifactTypes: T[Set[Type]] = Task { coursier.core.Resolution.defaultTypes }
+  def artifactTypes: T[Set[Type]] =
+    Task { coursier.core.Resolution.defaultTypes ++ Set(Type.Exotic.aar) }
```

Comment:
we need to avoid touching this, can't we change it in AndroidModule? it shouldn't be needed